### ISSUE TITLE
NVDA can't announce ComboBox collapse item change by arrows when DropDownStyle set as "DropDown"

### ISSUE THREAD

**Issue by MelonWang1** [dotnet/winforms#8395]
### .NET version

.NET SDK 8.0.100-alpha.1.22615.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it's not a regression issue, also repro on .NET 6.0.

### Issue description

NVDA has switch focus back to ComboBox after its dropdown is closed. But when close dropdown and Press Down: change selection to item "3", Press Down: change selection to item "4", the Speech Viewer don't show 3, 4.
![8220-1](https://user-images.githubusercontent.com/94418985/208012468-e0100b4c-c536-47e9-9b5b-e55f1e95430f.gif)

### Steps to reproduce

1.Extract the test app [ComboBox test.zip](https://github.com/dotnet/winforms/files/10242293/ComboBox.test.zip), then build and run it.
2.Launch NVDA screen reader
3.Expand ComboBox with keyboard (alt + down shortcut)
4.Select item "1" from dropdown list, press Enter or Esc to close the dropdown
5.Press Down: change selection to item "2"
   Press Down: change selection to item "3"
**Actual result:**
No announcement of value change by arrows.
![actual](https://user-images.githubusercontent.com/94418985/208012633-d7ba0f41-c6a8-40d9-8aee-8290e0f702b7.png)

**Expected result:**
NVDA can announce of value change by arrows.
![expected](https://user-images.githubusercontent.com/94418985/208012657-50f7a29e-0664-4c13-a97f-4cf19f39c328.png)

**More info:**
1.This issue cannot repro when set DropDownStyle set as "DropDownList".

**Comment by Cassie-Li01** [comment]
Verified in latest 8.0 build, issue is fixed. Please check below screenshot:
![image](https://user-images.githubusercontent.com/96758262/213405461-bbb371f1-9eae-4733-b4d6-694cf5ec7d08.png)

**Comment by Ashley-Li** [comment]
Verified this issue on .NET SDK 8.0.100-preview.1.23108.10 TP build, it was fixed that NVDA can announce of value change by arrows. Screenshot as above.
